src/main/java/frc/robot/subsystems/swerve/ModuleIOSim.java

```diff
@@ -67,4 +75,18 @@ public void setTurnVoltage(double volts) {
     turnAppliedVolts = MathUtil.clamp(volts, -12.0, 12.0);
     turnSim.setInputVoltage(turnAppliedVolts);
   }
+
+  @Override
+  public void setDriveSetpoint(double metersPerSecond) {
+    setDriveVoltage(
+        driveFeedforward.calculate(metersPerSecond)
+            + driveController.calculate(
+                driveSim.getAngularVelocityRadPerSec() * Module.WHEEL_RADIUS, metersPerSecond));
+  }
+
+  @Override
+  public void setTurnSetpoint(Rotation2d rotation) {
+    setTurnVoltage(
+        turnController.calculate(turnSim.getAngularPositionRotations(), rotation.getRotations()));
```

Comment:
ending parenthesis takes place on line 91

Reply:
spotless doesnt like that